README.md

```diff
@@ -626,6 +626,25 @@ An up-to-date list of the convenience operations can be found [here](https://doc
 | GET    | /v1/contextTypes                                                                     | R |
 | GET    | /v1/contextTypes{typename}                                                           | R ]
 
+#### NGSIv2 
+| Method | Path                                                                                    | Action |
+| ------ |:--------------------------------------------------------------------------------------- | ---:|
+| GET    | /v2                                                                                     | R |
+| GET    | /v2/entities                                                                            | R |
+| GET    | /v2\/entities/{entityId}                                                                | R |
+| POST   | /v2\/entities                                                                           | C |
+| PATCH  | /v2\/entities/{entityId}                                                                | U |
+| POST   | /v2\/entities/{entityId}                                                                | U |
+| DELETE | /v2\/entities/{entityId}                                                                | D |
+| PUT    | /v2\/entities/{entityId}                                                                | U |
+| GET    | /v2\/entities/{entityId}/attrs/{attrId}                                                 | R |
+| PUT    | /v2\/entities/{entityId}/attrs/{attrId}                                                 | U |
+| DELETE | /v2\/entities/{entityId}/attrs/{attrId}                                                 | D |
+| GET    | /v2\/entities/{entityId}/attrs/{attrId}/value                                           | R |
+| PUT    | /v2\/entities/{entityId}/attrs/{attrId}/value                                           | U |
+| GET    | /v2\/types                                                                              | R |
+| GET    | /v2\/types/{typeId}                                                                     | R |
+
```

Comment:
You are right, I forgot to include them in the documentation. I'll include them.